README.md

```diff
@@ -2,70 +2,109 @@
 
-The use of any Auro custom element has a dependency on the [Auro Design Tokens](https://auro.alaskaair.com/getting-started/developers/design-tokens).
+Defining the component dependency within each component that is using the `<auro-hyperlink>` component.
 
-### CSS Custom Property fallbacks
+```javascript
+import "@aurodesignsystem/auro-hyperlink";
+```
 
-[CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) are [not supported](https://auro.alaskaair.com/support/custom-properties) in older browsers. For this, fallback properties are pre-generated and included with the npm.
+### CDN Use
 
-Any update to the Auro Design Tokens will be immediately reflected with browsers that support CSS custom properties, legacy browsers will require updated components with pre-generated fallback properties.
+In cases where the project is not able to process JS assets, there are pre-processed assets available for use.
 
-### Define dependency in project component
+#### Install
 
-Defining the component dependency within each component that is using the `<auro-hyperlink>` component.
+Include the following HTML code in the `<head>` element of your page.
 
-```javascript
-import "@aurodesignsystem/auro-hyperlink";
+```html
+<link rel="stylesheet" href="https://unpkg.com/@alaskaairux/design-tokens@latest/dist/tokens/CSSCustomProperties.css" />
+<link rel="stylesheet" href="https://unpkg.com/@alaskaairux/webcorestylesheets@latest/dist/bundled/essentials.css" />
+
+<script src="https://unpkg.com/@aurodesignsystem/auro-hyperlink@latest/dist/auro-hyperlink__bundled.js" type="module"></script>
 ```
 
-**Reference component in HTML**
+**NOTE:** Be sure to replace `@latest` in the URL with the version of the asset you want. @latest is NOT aware of any MAJOR releases, use at your own risk.
+
+### Using the custom element
+
+After the web component is imported by JavaScript or CDN, reference the component in HTML.
+
 <!-- AURO-GENERATED-CONTENT:START (CODE:src=./apiExamples/basic.html) -->
 <!-- The below code snippet is automatically added from ./apiExamples/basic.html -->
 ```html
 <auro-hyperlink>Hello World</auro-hyperlink>
 ```
 <!-- AURO-GENERATED-CONTENT:END -->
 
-## Install bundled assets from CDN
+Review the <a href="https://auro.alaskaair.com/components/auro/hyperlink/api">API documentation</a> for advanced uses and examples.
 
-In cases where the project is not able to process JS assets, there are pre-processed assets available for use.
+### Limitations and Risks
 
-### Bundle example code
+When multiple microsites are combined and result in a custom element being imported more than once, only the first instance of the imported custom element is used. All subsequent attempts to load the custom element will fail. If a microsite loaded into the document subsequently attempts to load a new version of the component with an updated API, it will fail to load. This can result in unexpected API functionality.
```

Comment:
```suggestion
When multiple micro frontends are combined and result in a custom element being imported more than once, only the first instance of the imported custom element is used. All subsequent attempts to load the custom element will fail. If a micro frontend loaded into the document subsequently attempts to load a new version of the component with an updated API, it will fail to load. This can result in unexpected API functionality.
```

"Microsites" is not really a term. It's an internally invented phrase. The more common reference is micro frontends.